policy/README.md

```diff
@@ -352,9 +351,9 @@ An individual `Rule` object is defined by the following fields:
 [Top][toc]
 
 ### Rates
-Rate-related properties can currently be specified on `rate` and `time` Rules. Note: A future MDS version will likely support rates for `count` and `speed` rules, but their behavior is currently undefined.
+Rate-related properties can currently be specified on any policy with a `rate_amount`.   
 
-**[Beta feature](/general-information.md#beta-features)**: *Yes (as of 1.0.0)*. [Leave feedback](https://github.com/openmobilityfoundation/mobility-data-specification/issues/674)  
```

Comment:
If we do bring Rates out of beta, then this will need to read 

`**[Beta feature](/general-information.md#beta-features)**: *No (as of 2.0.0)*.`